inference-engine/tests_deprecated/unit/engines/vpu/adjust_data_batch_tests.cpp

```diff
@@ -346,3 +346,203 @@ TEST_F(VPU_AdjustDataBatchTest, Case5_MergeSubGraphs) {
     const auto& data14 = singleElement(checkSingleLoopEnd(data13));
     ASSERT_EQ(data14, testModel.getOutputs().at(0));
 }
+
+TEST_F(VPU_AdjustDataBatchTest, Case6_LinearWithBatchedInTheEnd) {
```

Comment:
`tests_deprecated` folder was considered as 'read-only' and contributions are restricted. Please add a new test cases to the `tests` folder even if it requires moving existing test

Reply:
@mikhail-treskin I don't think that adding extra case to deprecated tests is a reason to prevent tests from been added at all. Generally you're right, but in the context of coming FF/CF, I think it's ok.

@DariaMityagina I see you decided to get rid of tests, please raise a ticket with tech-debt label to enable it in the future or if @mikhail-treskin will agree just leave test as is and move tests to the new infra in the future.

Reply:
@gladilov-gleb alright. Will do.

Reply:
My general proposal is following - in most cases, moving of unit tests will require just moving sources to another folder maybe along with moving some utils. If so, please strive to move them in any case when you are extending `tests_deprecated` scope but not just modifying already existing tests e.g. to pass after some fixes in plugins. 

If moving will require more complex refactoring, I agree to have tech-debt ticket for the future. But I'm pretty sure that such tickets almost always will be deprioritized in dev teams in favor of more urgent tasks.

Reply:
I'll create ticket because this task is not completely done and I will continue to work on it (and on the ticket as part of the task). Therefore, it will not be deprioritized.